### PR TITLE
Initial version of Altoros Jenkins for PCF docs

### PR DESCRIPTION
**Pull request by Infra-Red**

### CODE REVIEW

**Comment by jncd** [comment]
Hello @Infra-Red I have done a copyedit and merge for your first PR. 
Make sure make a PR add the brief description of your product to the first page as indicated by the template. 
After you've done that, we can build it into a small app for you to review. It's looking good.
Thanks!

File: docs-content/index.html.md.erb [hunk 1]
```diff
@@ -17,59 +17,39 @@ owner: Partners
...
-<dt>YOUR-PRODUCT-NAME details:</dt>
-<dd><strong>Version</strong>: vYOUR-VERSION-NUMBER </dd>
-<dd><strong>Release date</strong>: YOUR-RELEASE-DATE (Ex: August 1, 2016)</dd>
-<dd><strong>Software component version</strong>: vYOUR-SOFTWARE-COMPONENT-VERSION-NUMBER</dd>
-<dd><strong>Compatible Ops Manager version(s)</strong>: vYOUR-COMPATIBLE-OPS-MAN-VERSION (Ex. v1.6.x, v1.7.x)</dd>
-<dd><strong>Compatible Elastic Runtime version(s)</strong>: vYOUR-COMPATIBLE-ELASTIC-RUNTIME-VERSION (Ex. v1.4.x, v1.5.x, v1.6.x)</dd>
-<dd><strong>IaaS support:</strong> AWS?, Azure?, GCP?, OpenStack?, and vSphere?</dd>
+<dt>Altoros Jenkins for PCF details:</dt>
+<dd><strong>Version</strong>: v0.8.0 </dd>
+<dd><strong>Release date</strong>: April 5, 2017</dd>
```
Comment (ViktoriyaKotova) [inline docs-content/index.html.md.erb]: @Infra-Red please change the release date to 04/10 and make a PR

File: docs-content/release-notes.html.md.erb [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-**This topic should include the Release Notes for your product. For examples, see the 
-[GCP Service Broker Release Notes](http://docs.pivotal.io/gcp-sb/release-notes.html) or the 
-[ISS Knowtify Search Analytics Release Notes](http://docs.pivotal.io/knowtify/release.html).
-When you release a new version, add new release notes to this file, keeping the older notes below.**
+### v0.8.0
 
-### vCURRENT-VERSION-NUMBER (Ex. v1.0.1)
-
-**Release Date:** RELEASE-DATE (Ex. August 1, 2016)
+**Release Date:** April 5, 2017
```
Comment (ViktoriyaKotova) [inline docs-content/release-notes.html.md.erb]: @Infra-Red release date 04/10

Comment (jncd) [inline docs-content/release-notes.html.md.erb]: I've made this change, thanks!

File: docs-content/index.html.md.erb (hunk 1 above)
Comment (ViktoriyaKotova) [inline docs-content/index.html.md.erb]: Thank you!

File: docs-content/release-notes.html.md.erb (hunk 2 above)
Comment (ViktoriyaKotova) [inline docs-content/release-notes.html.md.erb]: Thanks!

**Pull request merged**